core/locale/main/values/strings_ui.xml

```diff
@@ -69,6 +69,8 @@
     <string name='classes'>Classes</string>
     <string name='exams'>Exams</string>
     <string name='reports'>Reports</string>
+    <string name="choose_template">Choose template</string>
+    <string name="save_as_template">Save as \ntemplate</string>
```

Comment:
Why do we need the line break in the template here? Can't we handle that using the wrap options. This will be difficult for translators.